src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs

```diff
@@ -313,6 +315,72 @@ public TResponse Execute<TResponse>(OrganizationRequest request, string username
             return (TResponse)this.Execute(request);
         }
 
+        /// <inheritdoc/>
+        public bool HasStartedSolutionHistoryRecords(ILogger logger)
```

Comment:
The naming of this method doesn't seem to match what it is actually doing.

Reply:
@ewingjm refactored extended service method to better reflect its intent